indicators/NO_URGR_001-006/R/NO_URGR_001-006.qmd

```diff
@@ -0,0 +1,2547 @@
+URGR_006_tab_fylke <- fylke_soaa_imper_area %>%
+                        select(fylkesnumm, fylkesnavn, clc18_kode, soaa_area_m2,
+                                imper_area_mean, non_imper_area_mean) %>%
+                        group_by(fylkesnumm, fylkesnavn) %>%
+                        summarise(total_imper_area_m2 = sum(imper_area_mean),
+                                  total_non_imper_area_m2 = sum(non_imper_area_mean),
+                                  total_soaa_area_m2 = sum(soaa_area_m2))
+
+# Calculate summary table per region
+URGR_006_tab_fylke$fylkesnumm <- as.numeric(URGR_006_tab_fylke$fylkesnumm)
+URGR_006_tab_region <- URGR_006_tab_fylke %>%
+                          left_join(., 
+                                    admin_bd[, c("fylke_nr", "region_name")], 
+                                    join_by(fylkesnumm == fylke_nr)) %>%
+                          group_by(region_name) %>%
+                          summarise(total_imper_area_m2 = sum(total_imper_area_m2),
+                                    total_non_imper_area_m2 = sum(total_non_imper_area_m2),
+                                    total_cty_grp_area_m2 = sum(total_soaa_area_m2))
+```
+
+
+These table have been exported to the internal repository and will be used in the Results section to create an overall summary table comparing all URGR indicators.
+
+
+
+
+
+
+
+
+<!--# 
+
+Use this header for documenting the analyses. Put code in separate code chunks, and annotate the code in between using normal text (i.e. between the chunks, and try to avoid too many hashed out comments inside the code chunks). Add subheaders as needed. 
+
+Code folding is activated, meaning the code will be hidden by default in the html (one can click to expand it).
+
+Caching is also activated (from the top YAML), meaning that rendering to html will be quicker the second time you do it. This will create a folder inside you project folder (called INDICATORID_cache). Sometimes caching created problems because some operations are not rerun when they should be rerun. Try deleting the cash folder and try again.
+
+-->
+
+## 10. Results
+
+### National results
+```{r}
+#| eval: true
+#| code-summary: "Code to create a comparative URGR table"
+
+# Load the results files
+path_res <- "/data/P-Prosjekter2/412413_2023_no_egd/Results/URGR/"
+urgr001 <- read_excel(paste0(path_res, "URGR_001_detailed_results_24MAY25.xlsx"))
+urgr002 <- read_excel(paste0(path_res, "URGR_002_detailed_results_24MAY25.xlsx"))
+urgr003 <- read.table(paste0(path_res, "URGR_003_detailed_results_27MAY2025.csv"))
+urgr004 <- read.table(paste0(path_res, "URGR_004_detailed_results_26MAY2025.csv"))
+urgr005 <- read.table(paste0(path_res, "URGR_005_detailed_results_26JUNE2025.csv"))
+urgr006 <- read_excel(paste0(path_res, "URGR_006_fylke_results_18JUNE_25.xlsx"))
+
+# Calculate URGR indicators
+urgr001_national <- round2(sum(urgr001$total_area_et_m2) / sum(unique(urgr001$lau_area_m2)) *100, 2)
```

Comment:
The aggregation pathway is not clear to me. It seems like you have spatially aggregated the total urban area for Norway, and then you divide it by the total urban green area for Norway. Correct? You sum it in steps, but still that's where you end up. It might be fine, but then how would you calculate the uncertainty? Maybe the % urban green can be calulated for each city/polygon? Then we would get a distribution of variable values instead of just one.